incubator/quarkus/image/Dockerfile-stack

```diff
@@ -6,13 +6,13 @@ COPY ./project /project
 COPY ./config /config
 
 WORKDIR /project
-RUN mkdir "user-app"
 
 RUN mvn -N io.takari:maven:wrapper -Dmaven=$(mvn help:evaluate -Dexpression=maven.version -q -DforceStdout)
 RUN mvn -B install dependency:go-offline -DskipTests
 
-ENV APPSODY_MOUNTS="src:/project/user-app/src;pom.xml:/project/user-app/pom.xml;mvnw:/project/user-app/mvnw;mvnw.cmd:/project/user-app/mvnw.cmd"
+ENV APPSODY_MOUNTS=".:/project/user-app"
 ENV APPSODY_DEPS="/home/quarkus/.m2"
+ENV APPSODY_PREP="mkdir /project/user-app/target"
```

Comment:
This is odd. Does that work for you?
I got:
```
[Container] Running APPSODY_PREP command: mkdir /project/user-app/target
[Container] mkdir: cannot create directory '/project/user-app/target': Permission denied
[Container] [Error] FATAL error APPSODY_PREP command received an error.  The controller is exiting: exit status 1
```

Reply:
No, I didn't see that. I'm using a Mac. 

However, I am having trouble testing locally with the version upgrade commit (f4446a7555a2895e1a1181d108091108eb88669f). I updated the PR so as to test in CI, so as to see if my issue was reproduced, but it wasn't.

I get a different error locally, using this technique to use liocally: https://appsody.dev/docs/stacks/package/

```
paul@Pauls-Mac-mini:~/tmp/quarkus-stack/quarkus-stack# appsody run
Running development environment...
Using local cache for image dev.local/appsody/quarkus:0.4
Running command: docker run --rm -p 5005:5005 -p 8080:8080 --name quarkus-stack-dev -v /Users/paul/tmp/quarkus-stack/quarkus-stack/.:/project/user-app -v quarkus-stack-deps:/home/quarkus/.m2 -v appsody-controller-0.3.4:/.appsody -t --entrypoint /.appsody/appsody-controller dev.local/appsody/quarkus:0.4 "--mode=run"
[Container] Running APPSODY_PREP command: mkdir /project/user-app/target
[Container] Running command:  ./mvnw -B compile quarkus:dev
[Container] [INFO] Scanning for projects...
[Container] [INFO]
[Container] [INFO] ----------------------< org.acme:getting-started >----------------------
[Container] [INFO] Building getting-started 1.0-SNAPSHOT
[Container] [INFO] --------------------------------[ jar ]---------------------------------
[Container] [INFO]
[Container] [INFO] --- maven-resources-plugin:2.6:resources (default-resources) @ getting-started ---
[Container] [INFO] Using 'UTF-8' encoding to copy filtered resources.
[Container] [INFO] Copying 1 resource
[Container] [INFO]
[Container] [INFO] --- maven-compiler-plugin:3.1:compile (default-compile) @ getting-started ---
[Container] [INFO] Changes detected - recompiling the module!
[Container] [INFO] Compiling 2 source files to /project/user-app/target/classes
[Container] [INFO]
[Container] [INFO] --- quarkus-maven-plugin:1.3.0.Final:dev (default-cli) @ getting-started ---
[Container] Listening for transport dt_socket at address: 5005
[Container] Exception in thread "main" java.lang.RuntimeException: java.lang.RuntimeException: io.quarkus.bootstrap.BootstrapException: Failed to create the application model for org.acme:getting-started::jar:1.0-SNAPSHOT
[Container] 	at io.quarkus.dev.DevModeMain.main(DevModeMain.java:52)
[Container] Caused by: java.lang.RuntimeException: io.quarkus.bootstrap.BootstrapException: Failed to create the application model for org.acme:getting-started::jar:1.0-SNAPSHOT
[Container] 	at io.quarkus.dev.DevModeMain.start(DevModeMain.java:112)
[Container] 	at io.quarkus.dev.DevModeMain.main(DevModeMain.java:47)
[Container] Caused by: io.quarkus.bootstrap.BootstrapException: Failed to create the application model for org.acme:getting-started::jar:1.0-SNAPSHOT
[Container] 	at io.quarkus.bootstrap.BootstrapAppModelFactory.resolveAppModel(BootstrapAppModelFactory.java:306)
[Container] 	at io.quarkus.bootstrap.app.QuarkusBootstrap.bootstrap(QuarkusBootstrap.java:136)
[Container] 	at io.quarkus.dev.DevModeMain.start(DevModeMain.java:107)
[Container] 	... 1 more
[Container] Caused by: io.quarkus.bootstrap.resolver.AppModelResolverException: Failed to read descriptor of org.acme:getting-started:jar:1.0-SNAPSHOT
[Container] 	at io.quarkus.bootstrap.resolver.maven.MavenArtifactResolver.resolveDescriptorInternal(MavenArtifactResolver.java:239)
[Container] 	at io.quarkus.bootstrap.resolver.maven.MavenArtifactResolver.newCollectManagedRequest(MavenArtifactResolver.java:332)
[Container] 	at io.quarkus.bootstrap.resolver.maven.MavenArtifactResolver.resolveManagedDependencies(MavenArtifactResolver.java:290)
[Container] 	at io.quarkus.bootstrap.resolver.BootstrapAppModelResolver.doResolveModel(BootstrapAppModelResolver.java:168)
[Container] 	at io.quarkus.bootstrap.resolver.BootstrapAppModelResolver.resolveManagedModel(BootstrapAppModelResolver.java:143)
[Container] 	at io.quarkus.bootstrap.BootstrapAppModelFactory.resolveAppModel(BootstrapAppModelFactory.java:292)
[Container] 	... 3 more
[Container] Caused by: org.eclipse.aether.resolution.ArtifactDescriptorException: Failed to read artifact descriptor for org.acme:getting-started:jar:1.0-SNAPSHOT
[Container] 	at org.apache.maven.repository.internal.DefaultArtifactDescriptorReader.loadPom(DefaultArtifactDescriptorReader.java:301)
[Container] 	at org.apache.maven.repository.internal.DefaultArtifactDescriptorReader.readArtifactDescriptor(DefaultArtifactDescriptorReader.java:171)
[Container] 	at org.eclipse.aether.internal.impl.DefaultRepositorySystem.readArtifactDescriptor(DefaultRepositorySystem.java:255)
[Container] 	at io.quarkus.bootstrap.resolver.maven.MavenArtifactResolver.resolveDescriptorInternal(MavenArtifactResolver.java:233)
[Container] 	... 8 more
[Container] Caused by: org.apache.maven.model.resolution.UnresolvableModelException: No versions matched the requested parent version range '[0.4,0.5)'
[Container] 	at org.apache.maven.repository.internal.DefaultModelResolver.resolveModel(DefaultModelResolver.java:204)
[Container] 	at org.apache.maven.model.building.DefaultModelBuilder.readParentExternally(DefaultModelBuilder.java:1070)
[Container] 	at org.apache.maven.model.building.DefaultModelBuilder.readParent(DefaultModelBuilder.java:846)
[Container] 	at org.apache.maven.model.building.DefaultModelBuilder.build(DefaultModelBuilder.java:337)
[Container] 	at io.quarkus.bootstrap.resolver.maven.MavenModelBuilder.build(MavenModelBuilder.java:44)
[Container] 	at org.apache.maven.repository.internal.DefaultArtifactDescriptorReader.loadPom(DefaultArtifactDescriptorReader.java:292)
[Container] 	... 11 more
[Container] [INFO] ------------------------------------------------------------------------
[Container] [INFO] BUILD SUCCESS
[Container] [INFO] ------------------------------------------------------------------------
[Container] [INFO] Total time:  3.333 s
[Container] [INFO] Finished at: 2020-03-23T15:50:49Z
[Container] [INFO] ------------------------------------------------------------------------
[Container] The file watcher is not running because no APPSODY_RUN/TEST/DEBUG_ON_CHANGE action was specified or it has been disabled using the --no-watcher flag.
```